contrib/registry/zookeeper/register.go

```diff
@@ -42,6 +44,20 @@ func WithTimeout(timeout time.Duration) Option {
 	return func(o *options) { o.timeout = timeout }
 }
 
+// WithUser with registry user.
+func WithUser(user string) Option {
+	return func(o *options) {
+		o.user = user
+	}
+}
+
+// WithPassword with registry password.
+func WithPassword(password string) Option {
+	return func(o *options) {
+		o.password = password
+	}
+}
+
```

Comment:
Do user passwords always come together, so why don't we put them together？

Reply:
like this?
```go
// WithDigestACL with registry password.
func WithDigestACL(user string, password string) Option {
	return func(o *options) {
		o.user = user
		o.password = password
	}
}
```